clients/go/cmd/zbctl/internal/commands/createWorker.go

```diff
@@ -27,24 +27,25 @@ import (
 	"github.com/camunda-cloud/zeebe/clients/go/pkg/commands"
 	"github.com/camunda-cloud/zeebe/clients/go/pkg/entities"
 	"github.com/camunda-cloud/zeebe/clients/go/pkg/worker"
-	"github.com/camunda-cloud/zeebe/clients/go/pkg/zbc"
 	"github.com/spf13/cobra"
 )
 
 var (
-	createWorkerHandlerFlag        string
-	createWorkerNameFlag           string
-	createWorkerTimeoutFlag        time.Duration
-	createWorkerRequestTimeoutFlag time.Duration
-	createWorkerMaxJobsActiveFlag  int
-	createWorkerConcurrencyFlag    int
-	createWorkerPollIntervalFlag   time.Duration
-	createWorkerPollThresholdFlag  float64
+	createWorkerHandlerFlag       string
+	createWorkerNameFlag          string
+	createWorkerTimeoutFlag       time.Duration
+	createWorkerMaxJobsActiveFlag int
+	createWorkerConcurrencyFlag   int
+	createWorkerPollIntervalFlag  time.Duration
+	createWorkerPollThresholdFlag float64
+	createWorkerMaxJobsHandleFlag int
 
 	createWorkerHandlerArgs []string
 )
 
 // createWorkerCmd represents the createWorker command
+var jobsHandled = 0
```

Comment:
:question: Is this threadsafe? Does it need to be? I'm not sure how this works in Go :)